src/validator.cpp

```diff
@@ -29,9 +29,7 @@ limitations under the License.
 #include "libcellml/when.h"
 
 #include <cmath>
-
 #include <iostream>
```

Comment:
This include may not be even needed?

Reply:
Good point, I haven't checked, but I am going to in a minute.